packages/main/src/themes/Panel.css

```diff
@@ -81,7 +81,7 @@
 }
 
 .ui5-panel-content {
-	padding: 0.625rem 1rem 1.375rem 1rem;
```

Comment:
We have to add a css variable and specify the correct values for each theme as the metrics specified in the that change are valid only for horizon theme

Reply:
reply

Reply:
Done